charts/jindocache/templates/master/statefulset.yaml

```diff
@@ -186,6 +186,9 @@ spec:
               {{- end }}
           {{- end }}
           {{- end }}
+          {{- if .Values.master.volumeMounts }}
```

Comment:
I think not only master needs volumeMounts so does worker and fuse.

Reply:
no need